bin/internal/repackage_gradle_wrapper.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
Wonder how much easier this would be (to read), if written in Dart...

Reply:
Doesn't this also need a Windows version? We can't run bash scripts on Windows.

I second the suggestion to do all this in Dart, unless there's a reason we need to do this before we have downloaded the Dart toolchain.

Reply:
This is not something I expect anyone to have to do. It's only there in case we need to repackage a new version of the wrapper. It's certainly not something an end-user would do.